app/services/reports/monthly_usps_letter_requests_report.rb

```diff
@@ -0,0 +1,22 @@
+require 'login_gov/hostdata'
+
+module Reports
+  class MonthlyUspsLetterRequestsReport < BaseReport
+    REPORT_NAME = 'monthly-usps-letter-requests-report'.freeze
+
+    def call
+      daily_results = transaction_with_timeout do
+        ::LetterRequestsToUspsFtpLog.where(ftp_at: first_of_this_month..end_of_today)
+      end
+      totals = calculate_totals(daily_results)
+      save_report(REPORT_NAME, {total_letter_requests: totals,
+                                daily_letter_requests: daily_results}.to_json)
+    end
+
+    private
+
+    def calculate_totals(daily_results)
+      daily_results.inject(0) {|sum, rec| sum + rec['letter_requests_count'].to_i }
```

Comment:
Ruby's Array has had a `#sum` method [since at least 2.4](https://ruby-doc.org/core-2.4.1/Array.html#method-i-sum), so next time we could do:

```
daily_results.map { |rec| rec['letter_requests_count'].to_i }.sum(0)
```